### PR TITLE
Fix header content (make it work with lunr search)

### PR DESCRIPTION
**Pull request by gmarpons**

### CODE REVIEW

**Comment by gmarpons** [comment]
These changes are easier to apply here until the UI stabilizes, but at some point all files under `supplemental_files` could be moved into the UI repo.

File: antora-playbook.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 site:
   title: Decidim Docs
-  url: https://beta.docs.decidim.org
+  url: https://docs.decidim.org
```
Comment (andreslucena) [inline antora-playbook.yml:3]: :+1:

File: supplemental_ui/partials/header-content.hbs [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <header class="header" role="banner">
   <nav class="navbar">
     <div class="navbar-brand">
-      <a class="navbar-item" href="{{or site.url (or siteRootUrl siteRootPath)}}">{{site.title}}</a>
```
Comment (andreslucena) [inline supplemental_ui/partials/header-content.hbs:4]: Oh I actually didn't notice that the `supplemental_ui/partials`

**Comment by andreslucena** [comment]
> at some point all files under supplemental_files could be moved into the UI repo. 

Yes, I think that'd be better, I don't like having two partials that do the same thing... But this PR is perfect at the moment!

**Pull request merged**